extensions/amp-analytics/0.1/test/test-events.js

```diff
@@ -678,5 +683,17 @@ describes.realWin('Events', {amp: 1}, env => {
         });
       });
     });
+
+    describe('should create correct readyReportPromise', () => {
+      it('with viewer hidden', () => {
+        sandbox.stub(tracker.root, 'getViewer', () => {
+          return {
+            isVisible: () => {return false;},
+          };
+        });
+        const promise = tracker.createReportReadyPromise_();
+        return promise;
```

Comment:
Do this:

1. Remember stub instance above `const stub = sandbox.stub(...)`
2. In the `then` of this promise, check that this stub was called.